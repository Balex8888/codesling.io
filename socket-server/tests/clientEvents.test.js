const ioClient = require('socket.io-client/dist/socket.io.js');

const clientEvents = require('../src/clientEvents');

describe('All client events should be functions', () => {
  // Expects all clientEvents to be functions
  test('client.ready should be a function', () => {
    expect(typeof clientEvents.default['client.ready']).toMatchSnapshot();
  });

  test('client.update should be a function', () => {
    expect(typeof clientEvents.default['client.update']).toMatchSnapshot();
  });

  test('client.disconnect should be a function', () => {
    expect(typeof clientEvents.default['client.disconnect']).toMatchSnapshot();
  });
});
