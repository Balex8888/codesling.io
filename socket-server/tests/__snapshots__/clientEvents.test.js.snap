// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All client events should be functions client.disconnect should be a function 1`] = `"function"`;

exports[`All client events should be functions client.ready should be a function 1`] = `"function"`;

exports[`All client events should be functions client.update should be a function 1`] = `"function"`;
